runtime/doc/api.txt

```diff
@@ -1625,13 +1625,14 @@ nvim_exec({src}, {output})                                       *nvim_exec()*
                 On execution error: fails with VimL error, updates v:errmsg.
 
                 Parameters: ~
-                    {src}     Vimscript code
-                    {output}  Capture and return all (non-error, non-shell
-                              |:!|) output
+                    {src}   Vimscript code
+                    {opts}  Optional parameters.
+                            • output: Capture and return all (non-error,
+                              non-shell |:!|) output. Default: `false`.
```

Comment:
What flag should control the "stop on error" behavior? What's a common name for this kind of thing? (from anywhere, not just vim...).

`:help execute()` has a `"silent"` and `"silent!"` arg but that seems wrong for `nvim_exec2()` because we don't really want "silence" we want "continue on error".

Reply:
I like explicit names, so `stop_on_error = true` works for me.
Others off the top of my head: `safe = true`, `try_catch = false`.

Reply:
`continue=false`

Reply:
`continue` is a keyword in some programming languages, could that a problem?

Reply:
Not in Lua 5.1 :P

Reply:
>What's a common name for this kind of thing? (from anywhere, not just vim...).

Shell / bash has `set -o errexit` for "stop on error".

Reply:
Isn't the term used in vimscript `abort`?